DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/Index.cshtml

```diff
@@ -0,0 +1,136 @@
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@section NavBreadcrumbs {
+  <partial name="~/Views/TrackingSystem/Delegates/Shared/_AllDelegatesBreadcrumbs.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-two-thirds">
+    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3">@Model.DelegateInfo.Name</h1>
+  </div>
+  @if (Model.DelegateInfo.IsActive) {
+    <div class="nhsuk-grid-column-one-third view-delegate-top-button-group">
+      <a class="nhsuk-button nhsuk-button--secondary view-delegate-button" role="button">
+        Send welcome email
+      </a>
+    </div>
+  }
+
+  <div class="nhsuk-grid-column-full">
+    <div class="nhsuk-u-margin-bottom-5">
+      <strong class="@activeTagCss">@(Model.ActiveTagName)</strong>
+      <strong class="nhsuk-tag nhsuk-tag--grey">@(Model.RegStatusTagName)</strong>
+      <strong class="@passwordTagCss">@(Model.PasswordTagName)</strong>
+      @if (Model.DelegateInfo.IsAdmin) {
+        <strong class="nhsuk-tag nhsuk-tag--grey">Admin</strong>
+      }
+    </div>
+
+    <h2 class="nhsuk-heading-l nhsuk-u-margin-bottom-2">Details</h2>
+    <dl class="nhsuk-summary-list details-list-with-button">
+      <div class="nhsuk-summary-list__row details-list-with-button__row">
+        <dt class="nhsuk-summary-list__key">Name</dt>
+        <dd class="nhsuk-summary-list__value">@Model.DelegateInfo.Name</dd>
+      </div>
+
+      <div class="nhsuk-summary-list__row details-list-with-button__row">
+        <dt class="nhsuk-summary-list__key">Email</dt>
+        <dd class="nhsuk-summary-list__value">
+          @(Model.DelegateInfo.Email ?? "-")
+          @if (Model.DelegateInfo.Email == null) {
+            <span class="nhsuk-u-visually-hidden">No value set</span>
+          }
+        </dd>
+      </div>
+
+      <div class="nhsuk-summary-list__row details-list-with-button__row">
+        <dt class="nhsuk-summary-list__key">ID</dt>
+        <dd class="nhsuk-summary-list__value">@Model.DelegateInfo.CandidateNumber</dd>
+      </div>
+
+      <div class="nhsuk-summary-list__row details-list-with-button__row">
+        <dt class="nhsuk-summary-list__key">Alias</dt>
+        <dd class="nhsuk-summary-list__value">
+          @(Model.DelegateInfo.AliasId ?? "-")
+          @if (Model.DelegateInfo.AliasId == null) {
+            <span class="nhsuk-u-visually-hidden">No value set</span>
+          }
+        </dd>
+      </div>
+
+      <div class="nhsuk-summary-list__row details-list-with-button__row">
+        <dt class="nhsuk-summary-list__key">Registration date</dt>
+        <dd class="nhsuk-summary-list__value">
+          @(Model.DelegateInfo.RegistrationDate ?? "-")
+          @if (Model.DelegateInfo.RegistrationDate == null) {
+            <span class="nhsuk-u-visually-hidden">No value set</span>
+          }

```

Comment:
Could we change this in other places of the system too for consistency and add a comment about this decision on the ticket?